api/tools/build.gradle

```diff
@@ -52,6 +52,15 @@ task manageClusters(type: JavaExec) {
   }
 }
 
+task elasticSearchIndexer(type: JavaExec) {
+  classpath sourceSets.main.runtimeClasspath
+  main = "org.pmiops.workbench.tools.elastic.BigQueryOmopIndexer"
```

Comment:
nitpick: would it be possible to aim for consistent naming of the task / command-line / Java class? The other tools were mostly good about this.